.github/workflows/deploy-dev.yml

```diff
@@ -12,6 +12,32 @@ on:
       - success
 
 jobs:
+  deploy-csv-exporter:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [14.x, 16.x]
```

Comment:
I know it's in the example config for the action, but I'm pretty sure we _don't_ want to deploy twice with two different node versions.

Reply:
Would I be able to trial this, and if it works on both versions I'll remove 14 from the grid. (My suspicion is it won't work on 16)